recipes-crosswalk/crosswalk/crosswalk_20.50.533.12.bb

```diff
@@ -387,23 +414,25 @@ DEFAULT_CONFIGURATION = "\
     -Drelease_extra_cflags='-Wno-error=unused-local-typedefs' \
     -Dsysroot='' \
     -Ddisable_nacl=1 \
+    -Ddisable_bundled_extensions=1 \
     -Denable_printing=0 \
     -Dremoting=0 \
     -Duse_cups=0 \
+    -Duse_gconf=0 \
     -Duse_gio=0 \
     -Duse_gnome_keyring=0 \
     -Duse_kerberos=0 \
+    ${@bb.utils.contains('DISTRO_FEATURES', 'pulseaudio', '-Duse_pulseaudio=1', '-Duse_pulseaudio=0', d)} \
     -Duse_system_fontconfig=1 \
     -Duse_system_expat=1 \
-    -Duse_system_flac=1 \
     -Duse_system_harfbuzz=1 \
     -Duse_system_libevent=1 \
     -Duse_system_libjpeg=1 \
     -Duse_system_libpng=1 \
     -Duse_system_libusb=1 \
     -Duse_system_libxslt=1 \
-    -Duse_system_speex=1 \
     -Duse_system_yasm=1 \
+    -Dwerror='' \
```

Comment:
I think this will cover `-Drelease_extra_cflags='-Wno-error=unused-local-typedefs'` above. BTW, do you have any estimation on the effort to avoid adding this line?